packages/lesswrong/lib/collections/users/helpers.tsx

```diff
@@ -467,3 +467,8 @@ export const getAuth0Id = (user: DbUser) => {
   }
   throw new Error("User does not have an Auth0 user ID");
 }
+
+export const requireNewUserGuidelinesAck = (user: UsersCurrent) => {
```

Comment:
Why this implementation over having a migration? Also something something forum gating

Reply:
Moved the forum gating here - why a migration?